.github/workflows/sql-odbc-main.yml

```diff
@@ -61,19 +67,19 @@ jobs:
     #    cp ./bin64/*.log test-output
     - name: upload-build
       if: success()
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v3
       with:
         name: mac64-build
         path: sql-odbc/build-output
     - name: upload-installer
       if: success()
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v3
       with:
         name: mac64-installer
         path: sql-odbc/installer
     #- name: upload-test-results
-    #  if: success()
-    #  uses: actions/upload-artifact@v1
+    #  if: always()
```

Comment:
...? what's the point of this

Reply:
To upload test results even on failure.

Reply:
K.  If you want to keep the commented out code, please include a comment what it's useful for. 
Note: if you remove the comments, the git-history will have the changes and can be reverted.